### PR TITLE
Test Bunchbatch model

### PR DESCRIPTION
**Pull request by rocandres**
solved issues #152

### CODE REVIEW

File: harvests/tests/test_models.py [hunk 1]
```diff
@@ -53,3 +54,39 @@ def test_create_harvester(self):
...
+        new_bunchbatch.bunches.add(self.bunch);
+        new_bunchbatch.harvesters.add(self.harvester);
+        self.assertEqual( new_bunchbatch.harvesters.get(pk=10000).name, 'Andres Cabanzo')
+
+    def test_delete_bunchbatch(self):
+        self.bunchbatch.delete()
+        self.assertNotIn(self.bunchbatch, self.queryset)
+
+
+
```
Comment (danialeresbar) [inline harvests/tests/test_models.py:91]: Please create the update test for this model
Thanks :)

**Review by danialeresbar — COMMENTED** [review]
Please check my suggestions
Once you add the changes we can integrate it 😉

**Comment by rocandres** [comment]
The update test was created for this model

**Review by danialeresbar — APPROVED** [review]
Nice!

**Pull request merged**